tests/Microsoft.Data.DataFrame.Tests/BufferTests.cs

```diff
@@ -102,5 +121,45 @@ public void TestAppendMany()
             Assert.Equal(5, intColumn.NullCount);
             Assert.False(intColumn.IsValid(7));
         }
+
+        [Fact]
+        public void TestBasicArrowStringColumn()
+        {
+            StringArray strArray = new StringArray.Builder().Append("foo").Append("bar").Build();
+            ArrowStringColumn stringColumn = new ArrowStringColumn("String", strArray.ValueBuffer, strArray.ValueOffsetsBuffer, strArray.NullBitmapBuffer, strArray.Length, strArray.NullCount);
+            Assert.Equal(2, stringColumn.Length);
+            Assert.Equal("foo", stringColumn[0]);
+            Assert.Equal("bar", stringColumn[1]);
+        }
+
+        [Fact]
+        public void TestArrowStringColumnWithNulls()
+        {
+            string data = "joemark";
+            byte[] bytes = Encoding.UTF8.GetBytes(data);
+            Memory<byte> dataMemory = new Memory<byte>(bytes);
+            ReadOnlyDataFrameBuffer<byte> dataBuffer = new ReadOnlyDataFrameBuffer<byte>(dataMemory, 7);
+            Memory<byte> nullMemory = new byte[] { 13 };
```

Comment:
It may be more readable/understandable if the `13` was written in either hex or binary. You could do `0x0D` or `0b1101`.